rdr_service/offline/genomic_pipeline.py

```diff
@@ -327,7 +327,7 @@ def create_aw2f_manifest(feedback_record):
                               bucket_name=config.BIOBANK_SAMPLES_BUCKET_NAME,
                               ) as controller:
         controller.generate_manifest(GenomicManifestTypes.AW2F,
-                                     _genome_type=config.GENOME_TYPE_ARRAY,
+                                     _genome_type=None,
```

Comment:
Should we just default the genome_type arg to being None in the corresponding methods and remove this declaration?

`def generate_manifest(self, manifest_type, _genome_type=None, **kwargs)` and in 
`def generate_and_transfer_manifest(self, manifest_type, genome_type=None, **kwargs):`

Reply:
It actually is defaulted to None in the lookup method, but we could default it in the referenced methods. I'll make a note to do this in a non-hotfix PR.